sme_finance_application_schema/translations.py

```diff
@@ -27,35 +27,81 @@ def finance_application_v3_to_sme_contact_v3(finance_application):
 def finance_application_v3_to_sme_v5(finance_application):
     sme_v5 = {}
     for field in ('legal_status', 'months_revenue', 'revenue',
-        'sic_code', 'profitability', 'business_assets',
-        'overseas_revenue', 'exports', 'stock_imports', 'purchase_orders',
-        'directors_pensions', 'up_to_date_accounts', 'financial_forecast',
-        'business_plan', 'card_revenue', 'online_revenue', 'institutional_revenue',
-        'stock_ready', 'revenue_growth', 'intellectual_property', 'trade_credit',
-        'business_premises', 'registered_brand', 'customers', 'region',
-        'company_credit_rating', 'familiarity_with_financing', 'accounting_software'):
+                  'sic_code', 'profitability', 'business_assets',
+                  'overseas_revenue', 'exports', 'stock_imports', 'purchase_orders',
+                  'directors_pensions', 'up_to_date_accounts', 'financial_forecast',
+                  'business_plan', 'card_revenue', 'online_revenue', 'institutional_revenue',
+                  'stock_ready', 'revenue_growth', 'intellectual_property', 'trade_credit',
+                  'business_premises', 'registered_brand', 'customers', 'region',
+                  'company_credit_rating', 'familiarity_with_financing', 'accounting_software'):
         if field in finance_application['requesting_entity']:
             sme_v5[field] = finance_application['requesting_entity'][field]
-    for field in ( 'requested_amount', 'finance_type_requested', 'date_finance_required',
-            'date_finance_requested', 'finance_term_length', 'guarantor_available', 'purpose'):
+    for field in ('requested_amount', 'finance_type_requested', 'date_finance_required',
+                  'date_finance_requested', 'finance_term_length', 'guarantor_available', 'purpose'):
         if field in finance_application['finance_need']:
             sme_v5[field] = finance_application['finance_need'][field]
     if finance_application.get('actors'):
         directors = [x for x in finance_application['actors'] if x['role'] == 'director']
-        if directors:
-            if len(directors) > 1:
-                raise Exception
-            director = directors[0]
+        for director in directors:
             if 'value_of_property_equity' in director:
-                sme_v5['directors_houses'] = director['value_of_property_equity']
+                director_equity = ['value_of_property_equity']
+                if sme_v5.get('directors_houses'):
```

Comment:
maybe something like:

```
sme_v5.setdefault('directors_houses', 0)
sme_v5['directors_houses'] += director['value_of_property_equity']
```

makes it a little more compact, same for the pattern below in the pension section